tests/utils.py

```diff
@@ -112,8 +109,9 @@ def import_test_course(self):
         store = modulestore()
```

Comment:
From my above comment, I also tried using TestRuntime here as part of preparing runtime for the user. I was still able to get most of the tests to pass except for the 12 tests that were failing. You can try this as well if you want. If you go with this approach, you can create a `TestRuntime` in prepare runtime method and passing that to the runtime instead of `self.block.runtime`.

In this case: 12 tests that are failing are related to the fragment not being loaded properly in the student/studio view.

Reply:
The test cases which are passing do not rely on the runtime. The cases that are failing are the ones using the runtime. I also explored TestRuntime and discovered that it's simply a dummy class for getting the tests to run. In our case, we require a proper runtime to execute some tests.

Reply:
I agree. TestRuntime provides the basic runtime testing structure. This could however evolve over time whereas our hard-coded `definition_id` won't be. So, It would be better to use the basic skeleton of TestRuntime to keep things dynamically moving over the future.